src/client/utils/Intercom.tsx

```diff
@@ -47,3 +50,67 @@ export const Intercom: React.FC = () => {
 
   return null
 }
+
+export const IntercomHideOnScroll: React.FC = () => {
+  const intercomTimeout = useRef<null | NodeJS.Timeout>()
+  const paused = useRef<boolean>(false)
```

Comment:
Does `<boolean>` need to be specified when `false` is passed? I believe the type will be inferred anyway.